### ISSUE TITLE
Parsing of `"May"` yields `null` but other months succeed

### ISSUE THREAD

**Issue by honzatrtik** [wanasit/chrono#357]
Hello, is there any reason for following kind of behaviour?
```
> chrono.parseDate("January", undefined, { forwardDate: true })
2021-01-01T11:00:00.000Z
> chrono.parseDate("February", undefined, { forwardDate: true })
2021-02-01T11:00:00.000Z
> chrono.parseDate("March", undefined, { forwardDate: true })
2021-03-01T11:00:00.000Z
> chrono.parseDate("April", undefined, { forwardDate: true })
2021-04-01T10:00:00.000Z
> chrono.parseDate("May", undefined, { forwardDate: true })
null
> chrono.parseDate("June", undefined, { forwardDate: true })
2020-06-01T10:00:00.000Z
> chrono.parseDate("July", undefined, { forwardDate: true })
2020-07-01T10:00:00.000Z
> chrono.parseDate("September", undefined, { forwardDate: true })
2020-09-01T10:00:00.000Z
> chrono.parseDate("October", undefined, { forwardDate: true })
2020-10-01T10:00:00.000Z
> chrono.parseDate("November", undefined, { forwardDate: true })
2020-11-01T11:00:00.000Z
> chrono.parseDate("December", undefined, { forwardDate: true })
2020-12-01T11:00:00.000Z
```

Thanks in advance! ;-)

**Comment by honzatrtik** [comment]
https://github.com/wanasit/chrono/pull/359

**Comment by wanasit** [comment]
Thank you for letting me know.
I fixed this in 8c0c24efa5113326730cebd4538793e06cd74eb9.
